README.md

```diff
@@ -11,11 +11,14 @@ The source code is released under the MIT License.
     pip install python-rake
 
 ### Usage ###
+#for external .txt files
     import RAKE
-    Rake = RAKE.Rake([path_to_your_stopwords_file]);
-    # You can use one of the stoplists included in the repository under stoplists/
+    Rake = RAKE.Rake(<path_to_your_stopwords_file>);
+    Rake.run(text);
+#for built in stoplists
+    import RAKE
+    Rake = RAKE.Rake(<keyword>); #current keywords are SmartStopList and FoxStopList
```

Comment:
I'd wrap these in quotes, so its clearer to the user that the `<keyword>` is a string

~~~
RAKE.Rake(<keyword>); #current keywords are "SmartStopList" and "FoxStopList"
~~~

Otherwise as a user I'd be tempted to write

~~~
Rake = RAKE.Rake(SmartStopList)
~~~

Though arguably 

~~~
Rake = RAKE.Rake(RAKE.SmartStopList())
~~~

would be nice too?